h3/src/proto/frame.rs

```diff
@@ -389,9 +431,22 @@ setting_identifiers! {
     QPACK_MAX_TABLE_CAPACITY = 0x1,
     QPACK_MAX_BLOCKED_STREAMS = 0x7,
     MAX_HEADER_LIST_SIZE = 0x6,
+    // https://datatracker.ietf.org/doc/html/rfc9220#section-5
+    ENABLE_CONNECT_PROTOCOL = 0x8,
+    // https://datatracker.ietf.org/doc/html/draft-ietf-masque-h3-datagram-05#section-9.1
+    H3_DATAGRAM = 0xFFD277,
+    // https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-8.2
+    ENABLE_WEBTRANSPORT = 0x2B603742,
+    // https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-8.2
+    H3_SETTING_ENABLE_DATAGRAM_CHROME_SPECIFIC= 0xFFD277,
+
+    WEBTRANSPORT_MAX_SESSIONS = 0x2b603743,
+    // Dummy setting to check it is correctly ignored by the peer.
+    // https://datatracker.ietf.org/doc/html/rfc9114#section-7.2.4.1
+    DUMMY = 0x21,
```

Comment:
This is like the grease frames we send and receive, right? #92

Reply:
Looking now

Reply:
Dead code, probably I took it from aioquic but did not end up using it, I'm going to delete this.